Tests/GRDBTests/TableDefinitionTests.swift

```diff
@@ -63,18 +66,37 @@ class TableDefinitionTests: GRDBTestCase {
                 "c" TEXT, \
                 "d" BLOB, \
                 "e" ANY\
-                ) STRICT, WITHOUT ROWID
+                ) STRICT
                 """)
-            
-            do {
-                try db.execute(sql: "INSERT INTO test3 (id, a) VALUES (1, 'foo')")
-                XCTFail("Expected DatabaseError.SQLITE_CONSTRAINT_DATATYPE")
-            } catch DatabaseError.SQLITE_CONSTRAINT_DATATYPE {
-            }
```

Comment:
This test looks like it tests SQLite itself, and has little value.

Instead, let's say it tests that GRDB properly handles the `.strict` option, by enabling the side-effect that is intended by the user: SQLite throws an error when the application tries to insert a value with the wrong data type.

From this point of view, this test has a value. Would you please restore it in both `testStrictTableCreationOptionCustomAndCipher` and `testStrictTableCreationOption`?